Source/WebKit/WebProcess/cocoa/RemoteCaptureSampleManager.cpp

```diff
@@ -44,6 +44,8 @@
 namespace WebKit {
 using namespace WebCore;
 
+uint8_t RemoteCaptureSampleManager::RemoteAudio::s_realtimeThreadCount { 0 };
```

Comment:
shouldn't this be an atomic? 

Thie can be accessed on multiple threads at once without synchronisation.